observer-blindata-server/src/main/resources/application.yml

```diff
@@ -26,5 +26,6 @@ blindata:
     - eventType: DATA_PRODUCT_ACTIVITY_COMPLETED
       filter: ""
       activeUseCases:
+        - DATA_PRODUCT_VERSION_UPLOAD
```

Comment:
stage update must be a separate use case without uploading dp version if not needed... e.g if it is still in dev